src/components/Admin/AdminSearchForm.jsx

```diff
@@ -56,11 +56,10 @@ class AdminSearchForm extends React.Component {
           <div className="row w-100 m-0">
             <div className="col-12 col-md-3 px-0 pl-0 pr-md-2 pr-lg-3">
               <Form.Group>
-                <Form.Label id="course-title-search" className="search-label mb-2">Filter by course</Form.Label>
+                <Form.Label className="search-label mb-2">Filter by course</Form.Label>
                 <Form.Control
                   className="w-100"
                   as="select"
-                  aria-labelledby="course-title-search"
```

Comment:
am I correct in assuming the Paragon updates involve adding Aria labels to the Form.Control or Group component?

Reply:
It does! (you can see it in the snapshots) And it messes the labels if you pass them yourself, so... yeah.